src/runtime_src/core/pcie/linux/shim.cpp

```diff
@@ -1429,6 +1432,14 @@ int shim::xclReadTraceData(void* traceBuf, uint32_t traceBufSz, uint32_t numSamp
     return size;
 }
 
+
+#if 0
```

Comment:
if this is really needed for future use, please create a proper macro name - #ifdef XXX. Put #if 0 in multiple files for related code is not that clean.

Reply:
Done